docs/icd.md

```diff
@@ -103,58 +101,49 @@ Arrow services, which are written in Rust Programming Language, access gRPC feat
 
 ### :scroll: Files
 
-A `lib-common` repository will contain gRPC interfaces that will be used by every microservice.
+Every microservice will populate the following files:
 
 File | Description
 ---- | -----
-`common-grpc.proto` | Common gRPC interfaces, protocol buffer file.
-`common-grpc.rs` | Autogenerated Rust Code
-
-The `lib-common` crate will be published to a registry, and added as a dependency by each service:
-```toml
-# e.g. svc-scheduler Cargo.toml
-[dependencies]
-lib-common = 0.1.2
-```
+`proto/grpc.proto` | GRPC definitions used to autogenerate code stubs
+`server/src/grpc.rs` | Autogenerated Server Stubs
+`server/src/main.rs` | Implements Server Stubs
+`client-grpc/src/grpc.rs` | Autogenerated Client Code
 
-Service-specific gRPC files will be defined in each service's repository:
 ```tree
 .
-├── Cargo.lock
-├── Cargo.toml # [workspace] members = ['client', 'server']
+├── Cargo.toml
```

Comment:
\# [workspace] members = ['client-grpc', 'server']

And, how does this come together with the REST things above? Do we expect services like Cargo to have members = ['client-grpc', 'client-rest', 'server']?

Reply:
Updated to show that the REST microservice has a client-grpc as well

All services have grpc interfaces, some services have REST interfaces

https://github.com/Arrow-air/se-services/pull/5/commits/c86bba0ad757c2cf45365714b93776bb230e5429